src/index.js

```diff
@@ -19,8 +39,47 @@ function mapResult(attribute, keys, options, result) {
   });
 }
 
+function getCacheKey(model, attribute, options) {
+  return model.name + attribute + JSON.stringify(options, (key, value) => {
+    if (key === 'includeAssociation') {
+      return value.associationType + value.target.name + value.as;
+    }
+
+    return value;
+  });
+}
+
+function loaderForBTM(model, attributes, options = {}) {
+  assert(options.include === undefined, 'options.include is not supported by model loader');
+  assert(options.includeAssociation !== undefined, 'options.includeAssociation should be set for BTM loader');
+  assert(Array.isArray(attributes), 'Attributes for BTM loader should be an array');
+  assert(attributes.length === 2, 'Attributes for BTM loader should have length two');
+  assert(options.limit === undefined, 'BTM loader does not support limit');
```

Comment:
Actually, this would still need to be a join, so we can map users back to the projects